### PR TITLE
[System.XML] xunit tests and trivial types from CoreFX

### PR DESCRIPTION
**Pull request by EgorBo**
Adds 1192 xunit tests for System.XML from CoreFX
Also, moves common lines from `mobile_System.Xml.sources` and `System.Xml.sources` to `common.sources`

Part of #8132

### CODE REVIEW

File: mcs/class/System.XML/ReferenceSources/LocalAppContextSwitches.cs [hunk 1]
```diff
@@ -5,5 +5,7 @@ static class LocalAppContextSwitches {
 		public static readonly bool DontThrowOnInvalidSurrogatePairs = false;
 		public static readonly bool IgnoreKindInUtcTimeSerialization = false;
 		public static readonly bool EnableTimeSpanSerialization = false;
+		public static readonly bool LimitXPathComplexity = true;
+		public static readonly bool AllowDefaultResolver = true;
```
Comment (akoeplinger) [inline mcs/class/System.XML/ReferenceSources/LocalAppContextSwitches.cs]: are we sure these are the correct default values for us? aren't AppContext switches usually false and you need to use `true` to opt into the old/bad behavior?

File: mcs/class/System.XML/System.Xml.dll.sources [hunk 2]
```diff
@@ -586,87 +537,6 @@ ReferenceSources/BinaryCompatibility.cs
 ../referencesource/System.Xml/System/Xml/XmlUrlResolverAsync.cs
 ../referencesource/System.Xml/System/Xml/XmlUrlResolver.cs
 ../referencesource/System.Xml/System/Xml/XmlXapResolver.cs
-../referencesource/System.Xml/System/Xml/XPath/Internal/AbsoluteQuery.cs
```
Comment (akoeplinger) [inline mcs/class/System.XML/System.Xml.dll.sources]: any reason why those aren't replaced in mobile_System.Xml.dll.sources ?

File: mcs/class/System.XML/ReferenceSources/LocalAppContextSwitches.cs (hunk 1 above)
Comment (EgorBo) [reply mcs/class/System.XML/ReferenceSources/LocalAppContextSwitches.cs]: yeah, they should be `false` by default, thanks for noticing!

**Comment by EgorBo** [comment]
@monojenkins build failed

**Comment by marek-safar** [comment]
@monojenkins commit csproj

**Comment by marek-safar** [comment]
@monojenkins commit csproj

**Comment by marek-safar** [comment]
@EgorBo could you rebase the PR

**Comment by marek-safar** [comment]
@monojenkins commit csproj

**Comment by akoeplinger** [comment]
@monojenkins build Linux x64

**Comment by akoeplinger** [comment]
@monojenkins commit csproj

**Pull request merged**